### ISSUE TITLE
Error when trying to add voice circuit in 3.5.6 - serializernotfound

### ISSUE THREAD

**Issue by nikonau** [iDebugAll/phonebox_plugin#38]
**NetBox version**
3.5.6

**Describe the bug**
When trying to add a voice circuit (sip)
I was thrown this error:
`
<class 'netbox.api.exceptions.SerializerNotFound'>

Could not determine serializer for phonebox_plugin.VoiceCircuit with prefix ''

Python version: 3.8.10
NetBox version: 3.5.6
`

**To Reproduce**
In netbox 3.5.6 gui go to plugins > phonebox plugin > voice circuits and add a circuit - any combination yeilds above serializer error

**Expected behavior**
It should work

**Screenshots**
![error](https://github.com/iDebugAll/phonebox_plugin/assets/51310129/e6eab4d1-0776-4c8e-9b4c-a84e12a2c517)

**Comment by venkat5351** [comment]
I have the same issue too while configuring this plugin. Not sure about this. Do you have any inputs ?
![image](https://github.com/iDebugAll/phonebox_plugin/assets/141760237/4f96dfd2-1839-42c9-9087-50af1d1d522d)

**Comment by andersonit** [comment]
I'm having the same issue on my install recently updated to 3.5.8 and first time trying to use this plugin so I can't say if it worked before.  Any ideas are appreciated!
`<class 'netbox.api.exceptions.SerializerNotFound'>

Could not determine serializer for phonebox_plugin.VoiceCircuit with prefix ''

Python version: 3.10.12
NetBox version: 3.5.8
Plugins: 
  netbox_bgp: 0.10.2
  netbox_dns: 0.18.10
  netbox_documents: 0.6.0
  netbox_inventory: 1.4.2
  netbox_ipcalculator: 1.1
  netbox_ipv4_tools: 1.0.3
  netbox_otp_plugin: 1.0.7
  netbox_secrets: 1.8.5
  netbox_topology_views: 3.6.2
  phonebox_plugin: v0.0.6-beta.1
`

**Comment by venkat5351** [comment]
Nope. It didn't work for me as well. I am still looking if there is any official documentation available to troubleshoot.

**Comment by iDebugAll** [comment]
Hey Everyone! Sorry for such a delayed update. This issue is now fixed in the latest plugin release. Bonus: the plugin now also supports API operations with Voice Circuits. Compatibility has been tested with NetBox 3.6 versions.
